src/applications/disability-benefits/686c-674-v2/config/chapters/report-add-a-spouse/current-marriage-information/helpers.jsx

```diff
@@ -1,44 +1,141 @@
-      <p>
-        If an official record is not available, you should submit at least one
-        of the following documents:
-      </p>
-      <ul>
-        <li>
-          The official report from your branch of service showing a marriage
-          happened while you were in service
-        </li>
-        <li>A certified copy of the original marriage certificate</li>
-        <li>
-          An affidavit of the clergyman or magistrate who conducted the marriage
-          ceremony.
-        </li>
-        <li>Other evidence that may be considered</li>
-      </ul>
-    </va-additional-info>
+    <p>We’ll ask you to submit these documents at the end of this form.</p>
   </div>
 );
+
+const baseMessage = (
+  <>
+    <p>
+      Based on your answer, you’ll need to submit additional documents to add
+      your spouse as a dependent.
+    </p>
+    <p>We’ll ask you to submit these documents at the end of this form.</p>
+  </>
+);
+
+const typeProxy = (
+  <>
+    <p>
+      You’ll need to submit copies of all documents and certificates issued in
+      connection with your proxy marriage.
+    </p>
+  </>
+);
+
+const typeCommonLaw = (
+  <>
+    <p>You’ll need to submit copies of all these documents:</p>
+    <ul>
+      <li>
+        Copies of the birth certificates for you and your spouse’s children,{' '}
+        <strong>and</strong>
+      </li>
+      <li>
+        2 Statements of Marital Relationship (VA Form 21-4170)—1 that you
+        complete and 1 that your spouse completes, <strong>and</strong>
+        <br />
+        <va-link
+          href="/find-forms/about-form-21-4170/"
+          text="Get VA Form 21-4170 to
+          download (opens in new tab)"
+        />
+      </li>
+      <li>
+        2 Supporting Statements Regarding Marriage (VA Form 21P-4171) completed
+        by two different people with knowledge about your marriage.
+        <br />
+        <va-link
+          href="/find-forms/about-form-21p-4171/"
+          text="Get VA Form
+          21P-4171 to download (opens in new tab)"
+        />
+      </li>
+    </ul>
+  </>
+);
+
+const typeTribal = (
+  <>
+    <p>You’ll need to submit copies of all these documents:</p>
+    <ul>
+      <li>
+        Signed statements from you and your spouse. They must include the name
+        of the tribe, date (month, day, and year) of marriage, place (city and
+        state, county and state, or city and country) where the marriage
+        ceremony occurred, and the name and mailing address of the person who
+        performed the ceremony, <strong>and</strong>
+      </li>
+      <li>
+        Signed statements from at least two people who were present at the
+        tribal marriage ceremony. They must include the name of the tribe, date
+        (month, day, and year) of marriage, place (city and state, county and
+        state, or city and country) where the ceremony happened, and the name
+        and mailing address of the person who performed the ceremony,{' '}
```

Comment:
Just calling out I see a use of double and 
> "and the name and mailing address"

Wondering if we should bring this up with the larger team?

Reply:
This is per design
https://www.figma.com/design/7W55oNwdVXvXOTI9SaFzQ7/686c-Add-or-Remove-Dependents?node-id=1207-18113&t=UwoJDXEZ3DOCkmYA-0

Reply:
Yep, up to you if you want to circle back with the larger team. Just figured I would call it out 👍